packages/storage/src/task.ts

```diff
@@ -46,59 +46,47 @@ import {
   validate
 } from './implementation/args';
 import { async as fbsAsync } from './implementation/async';
-import { Location } from './implementation/location';
 import * as fbsMetadata from './implementation/metadata';
 import * as fbsRequests from './implementation/requests';
 import * as typeUtils from './implementation/type';
 import { Reference } from './reference';
-import { StorageService } from './service';
+import { getMappings } from './implementation/metadata';
 
 /**
  * Represents a blob being uploaded. Can be used to pause/resume/cancel the
  * upload and manage callbacks for various events.
  */
 export class UploadTask {
   private ref_: Reference;
-  private service_: StorageService;
-  private location_: Location;
-  private blob_: FbsBlob;
-  private metadata_: Metadata | null;
+  blob_: FbsBlob;
```

Comment:
use leading _ to be consistent across the SDK.

Reply:
Please mark the public but internal properties/methods with `@internal`

Reply:
Should be done.